src/types/filters-type.ts

```diff
@@ -0,0 +1,7 @@
+export type FiltersType = {
+  specialityName?: string;
+  serviceTypeName?: string;
+  name?: string;
+  firstAvailableAppointmentLabel?: string;
+  specialtyDegreeName?: string;
+};
```

Comment:
اسم‌های مناسب و مختصر انتخاب کنید

```suggestion
export type FiltersType = {
  speciality?: string;
  service?: string;
  query?: string;
  appointment?: string;
  degree?: string;
};
```